cranelift/codegen/src/verifier/flags.rs

```diff
@@ -28,13 +28,17 @@ pub fn verify_flags(
     errors: &mut VerifierErrors,
 ) -> VerifierStepResult<()> {
     let _tt = timing::verify_flags();
-    let mut verifier = FlagsVerifier {
-        func,
-        cfg,
-        encinfo: isa.map(|isa| isa.encoding_info()),
-        livein: SecondaryMap::new(),
-    };
-    verifier.check(errors)
+    if isa.is_none() || isa.unwrap().get_mach_backend().is_none() {
```

Comment:
Can you comment on why it makes sense to disable the FlagsVerifier with the mach backend?

Reply:
The FlagsVerifier as written uses the encoding info to determine whether instructions clobber the flags. We could certainly build an equivalent with the new infrastructure, or else add a target-independent notion of "clobbers flags" to the CLIF opcodes themselves, but it would be some extra plumbing. What do you think?

Reply:
The flags verifier can still check target-independent properties -- the `encinfo` is an `Option` to allow it to run without this information, so I think we can still enable the verifier, just by passing `None` for the `encinfo` argument when the mach backend is enabled.

Reply:
Ah, yes, that makes sense -- done!